lms/static/js/verify_student/views/webcam_photo_view.js

```diff
@@ -244,6 +247,17 @@
             // Install event handlers
             $( "#webcam_reset_button", this.el ).on( 'click', _.bind( this.reset, this ) );
```

Comment:
Why the 2 selectors? It's not clear from the file what this.el is set as? It is convention to set the el at the top of the Backbone View for clarity e.g. https://github.com/edx/edx-platform/blob/master/lms/static/js/student_account/views/RegisterView.js#L10
Can you please update to define el ?